tests/utils/test_transcript_annotation.py

```diff
@@ -0,0 +1,157 @@
+"""Tests for the transcript_annotation module."""
+
+import hail as hl
+import pytest
+
+from gnomad.utils.transcript_annotation import (
+    clean_tissue_name_for_browser,
+    create_tx_annotation_by_region,
+)
+
+
+class TestCleanTissueNameForBrowser:
+    """Tests for the clean_tissue_name_for_browser function."""
+
+    @pytest.mark.parametrize(
+        "input_name,expected_output",
+        [
+            ("BasalGanglia", "basal_ganglia"),
+            ("NucleusAccumbens", "nucleus_accumbens"),
+            ("SpinalCord", "spinal_cord"),
+            ("CervicalC", "cervical_c"),
```

Comment:
naive question, shouldn't one of these test the case where the entire word is lower case (like `basalganglia`)?